string  storage_path
string  name
---
bool    success
